src/pages/Public/NotFound/index.tsx

```
@@ -0,0 +1,146 @@
+import { useLocation, Link, useNavigate } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Button } from "@/components/ui/button";
+import { LanguageDropDown } from "@/components/LanguageDropDown";
+import { Home, ArrowLeft, Search, AlertTriangle } from "lucide-react";
+import logo from "@/assets/svgs/logo.svg";
+import bgAuth from "@/assets/svgs/authLayoutGradient.svg";
+import mainBgPurpleGradient from "@/assets/svgs/mainBgPurpleGradient.svg";
+import mainSecWorld from "@/assets/svgs/mainSecWorld.svg";
+import mainSecStar from "@/assets/svgs/mainSecStar.svg";
+
+interface NotFoundProps { }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)